app/views/posts/index.html.erb

```diff
@@ -14,6 +14,6 @@
     Recent posts
   </h3>
   <ul class="posts">
-    <%= render @timeline_posts %>
+    <%= render @posts %>
```

Comment:
You didn't address the previous reviewer's comment about validating the friendship. I can still see all users' posts even if I'm not friends with them.